.github/workflows/scenarios-permian.yml

```diff
@@ -117,6 +126,7 @@ jobs:
           ./pipeline --debug-log permian.log \
             -o library.directPath="${{ github.workspace }}/kickstart-tests/testlib" \
             -o workflows.dry_run=False \
+            -o kickstart_test.timeout="${{ steps.set_timeout.outputs.timeout }}" \
```

Comment:
Why are you not using the environment variable here directly? You could then drop the ``Set launcher timeout from job timeout`` step above.

Reply:
Yes, became too complicated. I wanted to have the timeout 20m less then the workflow timeout (so that the tests timeout before the workflow and we have partial logs). Because I am not able to use the timeout-minutes directly I wanted to be explicit about the workaround in the step... I think I will just set the value of TIMEOUT_MINUTES to the value (460) with a comment and use it directly here.

Reply:
I'm not sure you understand me. The point is not to use the matrix timeout value but to use the `env:` created variable here directly without the middle step.

```
env:
 TIMEOUT_MINUTES: 480

...

- name: Run scenario ${{ matrix.scenario }} in container
....
  -o kickstart_test.timeout="$(($TIMEOUT_MINUTES-20))m" \
```

Reply:
Yeah, but still, I'd do it more simple, see the commit message of https://github.com/rhinstaller/kickstart-tests/pull/697/commits/b5221032c76fa4ffb3ac3aa4afae30bbfec3c451I've added (may squash it).
I think I was using the step because I felt is wouldn't be so simple (would need to handle values less than 20 actually), but as I said, I'd do it simply now and use automation eventually in a separate PR.